script/release-workflow/docker-build.sh

```diff
@@ -2,6 +2,10 @@
 
 set -euo >/dev/null
 
-DOCKER_TARGET_PLATFORM=${DOCKER_TARGET_PLATFORM:-"linux/amd64"}
-
-docker buildx build --platform=${DOCKER_TARGET_PLATFORM} -t ${DOCKER_IMAGE_ORG_AND_NAME}:latest .
+for arch in arm64 arm amd64; do 
+    docker buildx build \
+    --platform linux/$arch \
+    --output type=docker \
```

Comment:
by using an output type of docker, we can load these in separate tests.

unfortunately you can't create multi-manifest builds ( by specifying platform values,  comma separated ) and load them into docker, they need to remain in the cache, or be pushed to a remote registry.

You can build them as a multi-arch build later however and it will use the images from the cache, in the `docker-tag.sh` step